evalme/image/object_detection.py

```diff
@@ -498,19 +498,20 @@ def compare(self, pred,
                                     gt_results_labels[0]['value']['labels'] == \
                                     pred_results_labels[0]['value']['labels']:
                                 # compare text results
-                                text_distance = self._compare_text_tags(pred_types=pred_types,
-                                                                        gt_results=gt_results,
-                                                                        pred_results=pred_results,
-                                                                        algorithm=algorithm,
-                                                                        qval=qval)
-                                if text_distance is None:
-                                    continue
-                            else:
-                                # in case of different labels or many labels
                                 text_tag_in_result = [item for item in pred_types if
                                                       item != 'labels' and item not in OCREvalItem.OCR_SHAPES]
                                 if not text_tag_in_result:
-                                    continue
+                                    text_distance = 1
```

Comment:
please check for the text result presence mismatch and set to 0 in this case

Reply:
Fixed